### ISSUE TITLE
OggOpus voice file frequency might not be 48000Hz

### ISSUE THREAD

**Issue by FrozenYogurtPuff** [FrozenYogurtPuff/Telegram-ASR-Bot#1]
## TL;DR
- Telegram voice might also be 16000Hz
- Use `ffprobe-python` or similar mediainfo package to ensure the frequency rate

## Background
Although the Telegram jni set OggOpus voice frequency to 48000Hz since [6.0.0](https://github.com/DrKLO/Telegram/blame/d30f796d8c912ce168f4e96d482c56cabd3c907e/TMessagesProj/jni/audio.c#L228) , yet the older version might [use an old frequency rate which is 16000Hz](https://github.com/DrKLO/Telegram/blame/df90eface5ebd755ea86c3a4eec66fe5b6e29041/TMessagesProj/jni/audio.c#L222) and there is an open PR warns that [the Android client still keeps at 16000Hz](https://github.com/DrKLO/Telegram/pull/1566).
